test/cpp/tensorexpr/test_llvm.cpp

```diff
@@ -824,7 +824,7 @@ void testLLVMComputeMul() {
     return Load::make(a, i, 1) * Load::make(b, i, 1);
   });
 
-  Buffer c_buf(VarHandle(c->function()->func_var()), kFloat32, {N});
+  Buffer c_buf(VarHandle(c->function()->func_var(c->output_index())), kFloat32, {N});
```

Comment:
A convenience wrapper from "c->func_var()" to "c->function()->func_var(c->output_index())" is useful. 

The way it calls now strongly suggests the underlying data structure, which breaks the software encapsulation.